docs/pages/installation.mdx

```diff
@@ -48,55 +48,391 @@ supports most features on Windows 10 and later.*
+agents by:
+- Following the instructions in the Teleport Web UI at `/web/discover`, where
+  you can select a resource to enroll in your Teleport cluster and retrieve an
+  installation script to run on Linux hosts.
+- Using the example Terraform module for deploying agents in [Deploy Agents with
+  Terraform](docs/pages/agents/deploy-agents-terraform.mdx). Read [Upgrade
+  Teleport Cloud Agents on Linux ](./upgrading/cloud-linux.mdx) to set up
+  automatic agent upgrades.
+- Reading the instructions in this section to install Teleport agent binaries
+  manually.
+
+If you are running a self-hosted Teleport Enterprise cluster, read our guidance
+in [Deploying a High-Availability Teleport
+Cluster](./deploy-a-cluster/introduction.mdx), provides full requirements and
+Terraform modules for deploying Teleport Enterprise.
+
+The instructions in this section are intended for:
+- Linux desktop users.
+- Operators who want to incorporate Teleport installation commands into custom
+  scripts when the approaches listed above are not suitable.
+
+### One-line installation script
+
+You can run a one-line command to install Teleport binaries on a Linux server:
 
-### Feature support
+```code
+$ EDITION="oss"
+$ VERSION="(=teleport.version=)"
+$ curl https://goteleport.com/static/install.sh | bash -s ${VERSION?} ${EDITION?} 
+```
 
-Some Teleport features have additional requirements:
+The command takes the Teleport version and edition to install, then uses
+information about the operating system where it runs to choose a package manager
+and install Teleport.
 
-| Feature                                                                        | Requirement   | Debian                   | Ubuntu   | CentOS/RHEL | Amazon Linux           | SLES           |
-|--------------------------------------------------------------------------------|---------------|--------------------------|----------|-------------|------------------------|----------------|
-| [Enhanced Session Recording](./server-access/guides/bpf-session-recording.mdx) | Kernel  v5.8+ | 11, or 10 with backports | 20.04.2+ | 9+          | 2 (post 11/2021), 2023 | 12 SP5, 15 SP5 |
-| Automatic Updates                                                              | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
-| Installation through apt/yum/zypper repos                                      | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
+Valid editions are:
 
-<Admonition type="note">
-`apt`, `yum`, and `zypper` repos don't expose packages for all distribution variants.
-When following installation instructions, you might need to replace `ID` with
-`ID_LIKE` to install packages of the closest supported distribution.
+| Value        | Edition                           |
+|--------------|-----------------------------------|
+| `cloud`      | Teleport Enterprise Cloud         |
```

Comment:
What does the script actually do when you specify "cloud" edition? We want all Cloud users to be using automatic upgrades, will this install the "teleport-ent-updater" as well?

Reply:
It currently does not. I'll open an issue in the repo where the one-line installation script lives so we can add this functionality.